plugins/pulp_rpm/plugins/importers/yum/repomd/metadata.py

```diff
@@ -294,6 +294,7 @@ def generate_dbs(self):
                 with contextlib.closing(gdbm.open(db_filename, 'nf')) as db_file_handle:
                     for element in generator:
                         utils.strip_ns(element)
+                        element.attrib['pkgid'] = '{{ pkgid }}'
```

Comment:
Should this use the template definition above in the PR? I don't know so I'm asking.